apps/heft/src/plugins/SassTypingsPlugin/SassTypingsGenerator.ts

```diff
@@ -30,6 +30,12 @@ export interface ISassConfiguration {
    */
   exportAsDefault?: boolean;
 
+  /**
+   * Files with these extensions will pass through the Sass transpiler for typings generation.
+   * Defaults to ["sass", "scss", "css"]
```

Comment:
For consistency with [this](https://github.com/microsoft/rushstack/blob/master/apps/heft/src/schemas/typescript.schema.json#L68-L74), let's require that these start with a `.`

Reply:
Can you also make sure that works with `typings-generator`. Maybe we even require that in `typings-generator`?

Reply:
The TypingsGenerator class appears to have logic to test for the `.` and add it if it's missing. 
Everything seems to work fine after adding the `.`